deepdow/experiments.py

```diff
@@ -215,6 +215,7 @@ def __init__(self, network, loss, train_dataloader, val_dataloaders=None, metric
         else:
             raise TypeError('Invalid type of benchmarks: {}'.format(type(benchmarks)))
 
+        # TODO: Add option to disable progress bar
```

Comment:
Legit point, are you suggesting this because the progress bar is buggy? Potentially related to #74

Reply:
What happens is that tqdm.tqdm don't work well on notebooks, so I'd recommend:

1. Option to disable
2. Option to use tqdm.notebook instead of tqdm (maybe a setup function in callbacks.py)

Reply:
I'll address all of the problems commented as soon as I can.

Reply:
I fixed the problem here, #80. Feel free to merge master into this branch to get the latest version.